gem_output.json

```diff
@@ -0,0 +1,302 @@
+{
+  "action_cable": [
+    "5.2.8.1",
+    "7.1.3.4",
+    "5.2.8.1",
+    "6.1.7.8"
+  ],
+  "action_mailer": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "6.1.7.8"
+  ],
+  "action_pack": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "7.0.8"
+  ],
+  "action_view": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "7.0.8"
+  ],
+  "active_job": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "6.1.7.8"
+  ],
+  "active_model_serializers": [
+    "0.10.13",
+    "0.10.14",
+    "0.10.13",
+    "0.10.13"
+  ],
+  "active_record": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "6.1.7.8"
+  ],
+  "active_support": [
+    "4.2.11.3",
+    "7.1.3.4",
+    "5.2.8.1",
+    "7.0.8"
+  ],
+  "aws": [
+    "3.181.0",
+    "3.199.0",
+    "3.181.0",
+    "3.181.0"
+  ],
+  "concurrent_ruby": [
+    "1.2.2",
+    "1.3.4",
+    "1.1.10",
+    "1.3.4"
+  ],
+  "dalli": [
+    "2.7.11",
+    "3.2.8",
+    "2.7.11",
+    "3.2.3"
+  ],
+  "delayed_job": [
+    "4.1.11",
+    "4.1.13",
+    "4.1.11",
+    "4.1.11"
+  ],
+  "elasticsearch": [
+    "7.17.11",
+    "8.17.0",
+    "7.17.11",
+    "8.17.0"
+  ],
+  "ethon": [
+    "0.16.0",
+    "0.16.0",
+    "0.14.0",
+    "0.16.0"
+  ],
+  "excon": [
+    "0.102.0",
+    "0.110.0",
+    "0.102.0",
+    "0.102.0"
+  ],
+  "faraday": [
+    "0.17.0",
+    "2.12.2",
+    "0.17.0",
+    "2.12.2"
+  ],
+  "grape": [
+    "1.7.0",
+    "2.1.2",
+    "1.7.0",
+    "1.8.0"
+  ],
+  "graphql": [
+    "1.13.21",
+    "2.3.7",
+    "1.13.21",
+    "2.3.6"
+  ],
+  "grpc": [
+    "1.48.0",
+    "1.67.0",
+    null,
+    null
+  ],
+  "hanami": [
+    "1.3.5",
+    "1.3.5",
+    null,
+    null
+  ],
+  "http": [
+    "5.0.1",
+    "5.2.0",
+    "4.4.1",
+    "4.4.1"
+  ],
```

Comment:
I am unsure if this is correct - I _think_ it is supposed to be `Net::HTTP` which is a library built into Ruby whereas `http.rb` is an actual Ruby gem.

So I think this one we may need to treat differently as it _should_ map to the tested Ruby versions of `Net::HTTP`

Thoughts?

Reply:
Ahh okay, so we essentially then have `http` and `http.rb` duplicated and are missing `net-http` in the table thanks!